src/ai/backend/install/context.py

```diff
@@ -509,6 +509,7 @@ async def configure_webserver(self) -> None:
                 if halfstack.redis_password:
                     redis_table["password"] = halfstack.redis_password
             data["session"]["redis"] = redis_table  # type: ignore
+            data["ui"]["menu_blocklist"] = "statistics,pipeline"  # type: ignore
```

Comment:
This line will add \`[ui].menu_blocklist` field with default value. Afterward, users can change the value from `webserver.conf`.